frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -44,26 +49,55 @@ export const quayURLFor = (vuln: ImageManifestVuln) => {
 
 export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = ({
   imageManifestVuln,
+  namespace,
 }) => {
+  const { t } = useTranslation();
   const resource = imageManifestVuln.data;
-
   const vulnsFor = (severity: string) =>
     resource.filter((v) => v.status?.highestSeverity === severity);
   const fixableVulns = resource
     .filter((v) => v.status?.fixableCount > 0)
     .reduce((all, v) => all.set(v.metadata.name, v), new Map<string, ImageManifestVuln>());
+  const getVulnerabilityCountText = (v: ImageManifestVuln, n?: string): string => {
+    const {
+      fixableCount = 0,
+      highCount = 0,
+      mediumCount = 0,
+      lowCount = 0,
+      unknownCount = 0,
+    } = v.status;
+    const totalCount = highCount + mediumCount + lowCount + unknownCount;
+    return n
+      ? t('container-security~{{fixableCount}} of {{totalCount}} fixable', {
```

Comment:
can make use of number formatter here and wherever we use number
```suggestion
      ? t('container-security~{{fixableCount, number}} of {{totalCount, number}} fixable', {
```

Reply:
done